consensus/common_test.go

```diff
@@ -367,3 +370,18 @@ func getSwitchIndex(switches []*p2p.Switch, peer *p2p.Peer) int {
 	panic("didnt find peer in switches")
 	return -1
 }
+
+// so we dont violate synchrony assumptions
```

Comment:
I would think that copying the test config every time is sufficient.  Why are we also resetting it?